packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/quillMarkup.tests.js

```diff
@@ -24,4 +24,16 @@ testModule('lists', () => {
             assert.strictEqual($listUi.css('visibility'), expectedVisibility, `UI element is ${hasUi ? '' : 'not'} visible`);
         });
     });
+
+    test('ordered list should have counter-reset property set to default (T1220554)', function(assert) {
```

Comment:
I believe we should test the correct behavior, not implementation details. Let's try to write a test with markup and check if the list ordering is correct. You can look at /tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/markup.tests.js for example ('render markup' test)